charts/hono/example/create_certs.sh

```diff
@@ -126,6 +126,14 @@ echo "trusted-ca.algorithm=$KEY_ALG" >> $DIR/trust-anchor.properties
 echo "trusted-ca.not-before=$NOT_BEFORE" >> $DIR/trust-anchor.properties
 echo "trusted-ca.not-after=$NOT_AFTER" >> $DIR/trust-anchor.properties
 
+echo "{" > default_tenant-trusted-ca.json
```

Comment:
the lines above where the variables are added to the `trust-anchor.properties` file seem to be not needed at all, or are they? If not, then I suggest to remove them ...